src/components/Button/Button.js

```diff
@@ -6,17 +6,21 @@ import HtmlElement from '../HtmlElement/HtmlElement';
 import { standard } from '../../themes';
 import { textMega } from '../../styles/style-helpers';
 
-const colorSelectorFor = baseColor => identifier => `${baseColor}${identifier}`;
+const sizes = {
+  KILO: 'kilo',
+  MEGA: 'mega',
+  GIGA: 'giga'
+};
```

Comment:
Should this instead go into some kind of constants file?

Reply:
You mean a file next to the component file? Could make sense, yeah. But we'd still add the sizes as properties to the Button component, right?